src/main/resources/config/liquibase/master.xml

```diff
@@ -122,6 +122,8 @@
     <include file="classpath:config/liquibase/changelog/20210519162242_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210528104100_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20210528163200_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20210605180001_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20210618160000_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
delete

Reply:
> Is there a reason why min/max % is still not restricted to 2 decimals while min/max points is? The provided precision is not needed I guess...
> 
> <img alt="Bildschirmfoto 2021-06-18 um 22 26 49" width="781" src="https://user-images.githubusercontent.com/41366522/122612947-4cf02880-d084-11eb-87f7-fdaa0c5422d8.png">

neither is restricted to 2 decimal points. I'm not rounding the values on this page, so the instructor is exactly aware of what the grades will be compared against. We wanted to keep the precision

Reply:
If I do that, there would be obsolete columns on the test servers and production